csharp/ql/test/query-tests/Security Features/CWE-502/UnsafeDeserializationUntrustedInput/UnsafeDeserializationUntrustedInputGood.cs

```diff
@@ -5,8 +5,8 @@ class Good
 {
     public static object Deserialize(TextBox textBox)
     {
-        JavaScriptSerializer sr = new JavaScriptSerializer();
+        JavaScriptSerializer sr = new JavaScriptSerializer(new SimpleTypeResolver());
         // GOOD
-        return sr.DeserializeObject(textBox.Text);
+        return sr.DeserializeObject("hardcoded");
```

Comment:
This file needs to stay in sync with the similar named file used by the `.qhelp` file for the query. Was the old example in fact not safe?